torch/_dynamo/symbolic_convert.py

```diff
@@ -1582,42 +1638,40 @@ def break_context_reference_cycle(val):
             val = val.call_function(self, [], {})  # type: ignore[arg-type]
 
         # Handle https://peps.python.org/pep-0479/
-        if (
-            is_generator(self.f_code)
-            and isinstance(val, variables.ExceptionVariable)
-            and val.exc_type is StopIteration
-        ):
-            val = variables.BuiltinVariable(RuntimeError).call_function(self, [], {})  # type: ignore[arg-type]
-
-        # set Exception.__context__
-        set_context_recursive(val, len(self.exn_vt_stack) - 1)
-        break_context_reference_cycle(val)
+        # if (
+        #     is_generator(self.f_code)
+        #     and isinstance(val, variables.ExceptionVariable)
+        #     and val.exc_type is StopIteration
+        # ):
+        #     val = variables.BuiltinVariable(RuntimeError).call_function(self, [], {})  # type: ignore[arg-type]
```

Comment:
I'm not sure if this is correct. This would convert any StopIteration by a RuntimeError at the point the exception was raised.